operator/apis/machinelearning.seldon.io/v1/seldondeployment_types.go

```diff
@@ -288,7 +289,8 @@ type SeldonPodSpec struct {
 	Metadata metav1.ObjectMeta `json:"metadata,omitempty" protobuf:"bytes,1,opt,name=metadata"`
 	Spec     v1.PodSpec        `json:"spec,omitempty" protobuf:"bytes,2,opt,name=spec"`
 	HpaSpec  *SeldonHpaSpec    `json:"hpaSpec,omitempty" protobuf:"bytes,3,opt,name=hpaSpec"`
-	Replicas *int32            `json:"replicas,omitempty" protobuf:"bytes,4,opt,name=replicas"`
+	PdbSpec  *SeldonPdbSpec    `json:"pdbSpec,omitempty" protobuf:"bytes,4,opt,name=pdbSpec"`
```

Comment:
I think we need to keep the order of elements to keep backward compatibility for gRPC protobufs.

Reply:
Fixed